spec/password_strategies/bcrypt_spec.rb

```diff
@@ -69,6 +69,26 @@ def encrypted_password
     end
   end
 
+  describe "#_encrypt" do
```

Comment:
As long as the existing tests pass, I'm happy to skip specific unit tests for the private method.